hw/ip/edn/doc/checklist.md

```diff
@@ -130,10 +130,10 @@ Regression    | [SIM_NIGHTLY_REGRESSION_SETUP][]      | Done        |
 Regression    | [FPV_REGRESSION_SETUP][]              | N/A         |
 Coverage      | [SIM_COVERAGE_MODEL_ADDED][]          | Done        |
 Integration   | [PRE_VERIFIED_SUB_MODULES_V1][]       | N/A         |
-Review        | [DESIGN_SPEC_REVIEWED][]              | Not Started |
-Review        | [DV_PLAN_REVIEWED][]                  | Not Started |
-Review        | [STD_TEST_CATEGORIES_PLANNED][]       | Not Started | Exception (?)
-Review        | [V2_CHECKLIST_SCOPED][]               | Not Started |
+Review        | [DESIGN_SPEC_REVIEWED][]              | Done        |
+Review        | [DV_PLAN_REVIEWED][]                  | Done        |
+Review        | [STD_TEST_CATEGORIES_PLANNED][]       | Done        | Exception (?)
```

Comment:
Can we get clarification on this item? Is it done, or is it exempt (if so, N/A instead of Done would fit better. And the question mark looks rather odd in a signed-off block.

Reply:
according to https://docs.opentitan.org/doc/project/checklist/#std_test_categories_planned, security, power and debug probably can be exception here. Performance may be not very critical in this module. we do have planned some stress cases, like fifo full, stress_all test.
@sriyerg any thoughts?

Reply:
I'm fine with having an exception here, let's just explicitly mention so and not leave a question in the checklist.

Reply:
I put in Security, Power, Debug